bin/quads.py

```diff
@@ -161,9 +179,11 @@ def main(argv):
     #
     #   force -  Some operations require --force.  E.g. if you want to redefine
     #            a cloud environment.
-
-    quads = Quads(args.config, args.statedir, args.movecommand, args.datearg,
-                  args.syncstate, args.initialize, args.force)
+    #
+    #   hardwareservice - ????
+    #
+    quads = libquads.Quads(args.config, args.statedir, args.movecommand, args.datearg,
```

Comment:
@portante one quick fix that got rid of some of the errors is to change all references to "libquads" to "Quads" as that is what the name of the package was changed to in the most recent upstream commits.